PendingReleaseNotes

```diff
@@ -38,3 +38,16 @@
   make the number of PGs per OSD exceed the configured
   ``mon_max_pg_per_osd`` limit.  The option can be adjusted if it
   is really necessary to create a pool with more PGs.
+
+* There was a bug in the PG mapping behavior of the new *upmap*
+  feature. If you made use of this feature (e.g., via the `ceph osd
+  pg-upmap-items` command), we recommend that all mappings be removed (via
+  the `ceph osd rm-pg-upmap-items` command) before upgrading to this
+  point release.
+
+13.0.0
```

Comment:
Nit: change this to 12.2.X (whatever the next release is) or add it to the relevant section if one already exists.

Reply:
ack.. scheduled tests anyway since the comments were only on the doc section which we can amend before merging